authoring-new-actions.md

```diff
@@ -0,0 +1,130 @@
+| **Visual Studio and .NET** | In Progress | Angelos Petropoulos |
+| **Visual Studio Code** | In Progress | TBD |
+| **Nuget** | [link](https://github.com/marketplace/actions/setup-nuget-exe-for-use-with-actions) | Anand Gaurav |
+
+Please reach of to us if your team is not listed above and would want to collaborate for new Actions.
+
+# Building a new GitHub Action by Microsoft
+
+## Repo Creation
+
+1. Ensure that only one action is present per repo. Create new repo for each of your actions.
+2. Choose the right org to create a repo. For example, for all Actions targeting an Azure service, choose [https://github.com/Azure/](https://github.com/Azure/) as the Org. Else for any generic utility actions which are cloud agnostic, chose [https://github.com/Microsoft](https://github.com/Microsoft) as the org for hosting the Action repo.
+3. Request a new repo from the below links as suitable:
+  1. [https://repos.opensource.microsoft.com/Azure](https://repos.opensource.microsoft.com/Azure) for creating repos under Azure Org
+  2. [https://repos.opensource.microsoft.com/microsoft](https://repos.opensource.microsoft.com/microsoft) for Microsoft Org
+4. Follow the Wizard for creating a &quot; **New release registration**&quot; and answer the below queries accordingly:
+  1. **Is this going to be a public open source project - Yes**
+  2. **What type of project is this? – Product Code (typically for all Actions)**
+  3. **What license will you be releasing it under? – MIT**
+  4. **Did your team write all the code and create all of the assets you are releasing? – Yes**
+  5. **Does this project send any data or telemetry back to Microsoft? – Yes, if your Action collects back any telemetry.**
+  6. **Provide a suitable Project name, version (can start with 0.0), description, business goals and Products where this will be used (GitHub workflows)**
+  7. **Give a suitable Repo name &amp; description (shows up as public-facing repo description in GitHub)**
+    1. Ensure that the repo name includes the exact verb/value that Action is intended to achieve.
+    2. For actions targeting Azure, ensure to include &quot;Azure&quot; key word in the repo name.
+  8. **Mark Initial repo visibility as**  **PRIVATE**
+  9. **Add** [actions-admin](https://repos.opensource.microsoft.com/Azure/teams/actions-admin) **as one of the**  **Administrators Teams who could manage repo settings. You can also add more teams if needed.**
+  10. **Choose License template (Generally all actions are open sourced under MIT license)**
+  11. **Review all the submitted details and create Request.**
+5. Note that this request triggers a Legal OSS approval process which needs to be completed before the repo visibility can be changed to PUBLIC. You should get an email from &quot;[noreply-opensource@microsoft.com](mailto:noreply-opensource@microsoft.com)&quot;  with a link to the work item that tracks the legal, OSS and your LT clearance.
```

Comment:
All of this seems like an internal process. Something which only an employee can relate to

Reply:
Removed the content.